mergify_engine/duplicate_pull.py

```diff
@@ -319,6 +321,9 @@ async def duplicate(
             "checking-out-pull-requests-locally/"
         )
 
+    if append_original_description and not ctxt.pull["body"] is None:
```

Comment:
I quickly submitted my draft PR but it's failing when running the flake8 with

```
TypedDict "GitHubPullRequest" has no key 'body'  [misc]
```

I somehow though the `body` attribute was a correct one. I'll need to debug further

Reply:
`body` is valid but it's just missing from the type definition in `github_types`.
Those definitions are not always complete (we're lazy). You can double-check GitHub API if you doubt.